### DISCUSSION TITLE
use sync over HTTP in .net 4.5

### DISCUSSION THREAD

**Discussion by guangyi1609** [Mimetis/Dotmim.Sync#411] [Q&A] [answered]
Hi the sync over HTTP can be using .net 4.5 as the example the provided is use .net core

**Comment by Mimetis** [comment, marked as answer]
Hi @guangyi1609
I'm sorry, it can't be used in classic .Net Framework.

Even if the `Dotmim.Sync.Web.Server` package is `.Net Standard 2.0` compatible, it's using some `ASP.Net Core` base libraries that are not compatible with `.Net Fx 4.7.2` 
Especially everything related to `HttpContext`.

If you are not able to create as `ASP.Net Core Web Api` project, I think it's still possible to create a `Dotmim.Sync.Web.Server.Fx472` project, based on the `Dotmim.Sync.Web.Server` source code, and adapted to `.Net Framework 4.7.2` (minimum .Net Framework version to be .Net Standard 2.0 compatible)
It's a lot of work I guess, but I'm convinced this is doable.

**Pull Requests** are open ! :)

**Comment by guangyi1609** [comment]
Thank for the reply I will start a ASP.Net Core Web Api
